scripts/data_files/query_config.fmt

```diff
@@ -10,72 +10,16 @@
 #include "query_config.h"
 
 #include "mbedtls/platform.h"
+#include <string.h>
 
-/*
- * Include all the headers with public APIs in case they define a macro to its
- * default value when that configuration is not set in mbedtls_config.h, or
- * for PSA_WANT macros, in case they're auto-defined based on mbedtls_config.h
- * rather than defined directly in crypto_config.h.
- */
-#include "psa/crypto.h"
-
-#include "mbedtls/aes.h"
-#include "mbedtls/aria.h"
-#include "mbedtls/asn1.h"
-#include "mbedtls/asn1write.h"
-#include "mbedtls/base64.h"
-#include "mbedtls/bignum.h"
-#include "mbedtls/camellia.h"
-#include "mbedtls/ccm.h"
-#include "mbedtls/chacha20.h"
-#include "mbedtls/chachapoly.h"
-#include "mbedtls/cipher.h"
-#include "mbedtls/cmac.h"
-#include "mbedtls/ctr_drbg.h"
-#include "mbedtls/debug.h"
-#include "mbedtls/des.h"
-#include "mbedtls/ecdh.h"
-#include "mbedtls/ecdsa.h"
-#include "mbedtls/ecjpake.h"
-#include "mbedtls/ecp.h"
-#include "mbedtls/entropy.h"
-#include "mbedtls/error.h"
-#include "mbedtls/gcm.h"
-#include "mbedtls/hmac_drbg.h"
-#include "mbedtls/md.h"
-#include "mbedtls/md5.h"
-#include "mbedtls/memory_buffer_alloc.h"
-#include "mbedtls/net_sockets.h"
-#include "mbedtls/nist_kw.h"
-#include "mbedtls/oid.h"
-#include "mbedtls/pem.h"
-#include "mbedtls/pk.h"
-#include "mbedtls/pkcs12.h"
-#include "mbedtls/pkcs5.h"
+/* Work around https://github.com/Mbed-TLS/TF-PSA-Crypto/issues/393 */
 #if defined(MBEDTLS_HAVE_TIME)
-#include "mbedtls/platform_time.h"
+#include <mbedtls/platform_time.h>
```

Comment:
Minor: all other program use `#include "mbedtls/foo.h"` with double quotes, I don't see a reason why this one should be different. (Same for the generated includes, which also makes a before-after comparison less straightforward.)

Reply:
Sorry, I hadn't paid attention to whether the original used `""` or `<>` when I wrote new code. (Even `platform_time.h` was new code because I removed it, then added it back in 409c688c4b595db2e178e805260fbfbbb9de5fd7.) I tend to use `<>` when the lookup is not intended to go through the current directory, so `<>` came naturally here.

I can go back and change the quotes again but at this point I guess it's not worth it.